tabular/src/autogluon/tabular/models/tabular_nn/tabular_nn_model.py

```diff
@@ -384,17 +384,18 @@ def train_net(self, train_dataset, params, val_dataset=None, initialize=True, se
         self.params_trained['num_epochs'] = best_val_epoch + 1
         return
 
-    def _predict_proba(self, X, **kwargs):
-        """ To align predict with abstract_model API.
-            Preprocess here only refers to feature processing steps done by all AbstractModel objects,
+    def _predict_proba(self, X, preprocess=True):
+        """ To align predict wiht abstract_model API.
+            Preprocess here only refers to feature processing stesp done by all AbstractModel objects,
             not tabularNN-specific preprocessing steps.
             If X is not DataFrame but instead TabularNNDataset object, we can still produce predictions,
             but cannot use preprocess in this case (needs to be already processed).
         """
         if isinstance(X, TabularNNDataset):
             return self._predict_tabular_data(new_data=X, process=False, predict_proba=True)
         elif isinstance(X, pd.DataFrame):
-            X = self.preprocess(X, **kwargs)
+            if preprocess:
```

Comment:
Revert

Reply:
This came in with the auto-merge, looks to be my CI breaking bug.